test/integration/data/sshkey/id_rsa

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
I don't want to have keys checked into version control.  I understand that they're just for the test, but you should generate them and remove them in the test as a self-contained unit.